django/db/backends/postgresql/features.py

```diff
@@ -51,6 +51,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_over_clause = True
     supports_aggregate_filter_clause = True
 
+    supported_explain_formats = {'TEXT', 'XML', 'JSON', 'YAML'}
```

Comment:
I think the style is not to have a blank line between the feature-flags, except in the base-backend. Please also see the MySQL feature flag.